### PR TITLE
ifconfig: correct placeholders

### LINKED ISSUE #14420: Page modification request: Ifconfigs placeholders are not properly formatted
### Command description

- Disable eth0 interface:

`ifconfig eth0 down`

- Enable eth0 interface:

`ifconfig eth0 up`

In the above commands eth0 can vary depending on the system and there are ways in which the default naming can be chnaged

### Command details

`ifconfig eth0 down`

### Documentation

https://net-tools.sourceforge.io/man/ifconfig.8.html

### Platform

Common

### VCS repository link (e.g. GitHub, GitLab)

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by vp007-dev**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**
closes #14420

### CODE REVIEW

File: pages/common/ifconfig.md [hunk 1]
```diff
@@ -3,22 +3,22 @@
 > Network Interface Configurator.
 > More information: <https://net-tools.sourceforge.io/man/ifconfig.8.html>.
 
-- View network settings of an Ethernet adapter:
+- View network settings of an Interface:
```
Comment (spageektti) [inline pages/common/ifconfig.md:6]:
```suggestion
- View network settings of an interface:
```

**Review by gutjuri — APPROVED** [review]

**Comment by spageektti** [comment]
One suggestion is still not resolved

File: pages/common/ifconfig.md [hunk 2]
```diff
@@ -3,22 +3,22 @@
...
+`ifconfig {{interface_name}} down`
 
-- Enable eth0 interface:
+- Enable interface:
 
-`ifconfig eth0 up`
+`ifconfig {{interface_name}} up`
 
-- Assign IP address to eth0 interface:
+- Assign IP address to interface:
```
Comment (acuteenvy) [inline pages/common/ifconfig.md:22]:
```suggestion
- Assign an IP address to an interface:
```

File: pages/common/ifconfig.md [hunk 3]
```diff
@@ -3,22 +3,22 @@
...
 `ifconfig -a`
 
-- Disable eth0 interface:
+- Disable interface:
 
-`ifconfig eth0 down`
+`ifconfig {{interface_name}} down`
 
-- Enable eth0 interface:
+- Enable interface:
```
Comment (acuteenvy) [inline pages/common/ifconfig.md:18]:
```suggestion
- Enable an interface:
```

File: pages/common/ifconfig.md [hunk 4]
```diff
@@ -3,22 +3,22 @@
...
 
-`ifconfig eth0`
+`ifconfig {{interface_name}}`
 
 - Display details of all interfaces, including disabled interfaces:
 
 `ifconfig -a`
 
-- Disable eth0 interface:
+- Disable interface:
```
Comment (acuteenvy) [inline pages/common/ifconfig.md:14]:
```suggestion
- Disable an interface:
```

**Review by acuteenvy — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**